internal/deploy/elastic-package.go

```diff
@@ -0,0 +1,331 @@
+	})
+	defer span.End()
+
+	return nil
+}
+
+// Remove remove services from deployment
+func (ep *EPServiceManager) Remove(ctx context.Context, profile ServiceRequest, services []ServiceRequest, env map[string]string) error {
+	span, _ := apm.StartSpanOptions(ctx, "Removing services from Elastic Package deployment", "elastic-package.services.remove", apm.SpanOptions{
+		Parent: apm.SpanFromContext(ctx).TraceContext(),
+	})
+	span.Context.SetLabel("profile", profile)
+	span.Context.SetLabel("services", services)
+	defer span.End()
+
+	if profile.Name != "fleet" {
+		return fmt.Errorf("profile %s not supported in elastic-package provisioner. Services: %v", profile.Name, services)
+	}
+
+	for _, service := range services {
+		manifest, inspectErr := ep.Inspect(context.Background(), service)
+		if inspectErr != nil {
+			log.Warnf("Service %s could not be deleted: %v", service.Name, inspectErr)
+			continue
+		}
+
+		_, err := shell.Execute(ep.Context, ".", "docker", "rm", "-fv", manifest.Name)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// Start a container
+func (ep *EPServiceManager) Start(ctx context.Context, service ServiceRequest) error {
+	span, _ := apm.StartSpanOptions(ctx, "Starting service from Elastic Package deployment", "elastic-package.service.start", apm.SpanOptions{
+		Parent: apm.SpanFromContext(ctx).TraceContext(),
+	})
+	span.Context.SetLabel("service", service)
+	defer span.End()
+
+	manifest, _ := ep.Inspect(context.Background(), service)
+	_, err := shell.Execute(ep.Context, ".", "docker", "start", manifest.Name)
+	return err
+}
+
+// Stop a container
+func (ep *EPServiceManager) Stop(ctx context.Context, service ServiceRequest) error {
+	span, _ := apm.StartSpanOptions(ctx, "Stopping service from Elastic Package deployment", "elastic-package.service.stop", apm.SpanOptions{
+		Parent: apm.SpanFromContext(ctx).TraceContext(),
+	})
+	span.Context.SetLabel("service", service)
+	defer span.End()
+
+	manifest, _ := ep.Inspect(context.Background(), service)
+	_, err := shell.Execute(ep.Context, ".", "docker", "stop", manifest.Name)
+	return err
+}
+
+func buildElasticAgentRequest(srv ServiceRequest, env map[string]string) tc.ContainerRequest {
+	privileged := false
+	var bindMounts map[string]string
+	var entrypoint []string
+	imageNamespace := fmt.Sprintf("elastic-agent%s", env["elasticAgentDockerImageSuffix"])
+	img := fmt.Sprintf("docker.elastic.co/%s/%s:%s", env["elasticAgentDockerNamespace"], imageNamespace, env["elasticAgentTag"])
+
+	if srv.Flavour == "centos" || srv.Flavour == "debian" {
+		imageNamespace = fmt.Sprintf("elastic-agent-%s", srv.Flavour)
+		// use observability's systemd base images to install the elastic-agent on them
+		img = "docker.elastic.co/observability-ci/" + srv.Flavour + "-systemd:latest"
+		entrypoint = []string{"/sbin/init"}
+		bindMounts = map[string]string{
+			"/sys/fs/cgroup": "/sys/fs/cgroup",
+		}
+		privileged = true
+	} else if srv.Flavour == "cloud" {
```

Comment:
I'm adding cloud flavour, but I think we can remove it, including the descriptors in the config dir